### PR TITLE
Announce letter price increases

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR adds an announcement to the pricing page of forthcoming price increases.

It follows the same pattern we used in 2021, with slightly updated content.

### CODE REVIEW

File: app/templates/views/pricing/index.html [hunk 1]
```diff
@@ -266,4 +266,13 @@ <h2 class="heading-medium" id="letters">Letters</h2>
...
   </div>
 
+  <div class="panel panel-border-wide">  
+    <p class="govuk-body">From Monday 23 January 2023:</p>
+    <ul class="list list-bullet">
+    <li>second class postage will go up by 6 pence, plus VAT</li>
+    <li>first class postage will go up by 9 pence, plus VAT</li>
+    <li>international postage will go up by 9 pence, plus VAT</li>
+  </ul>
+  </div>
```
Comment (samuelhwilliams) [inline app/templates/views/pricing/index.html:276]: The class names on this aren't quite right - they should be `govuk-list` and `govuk-list--bullet`. Currently on the page it looks like this, with the bullets off to the left.

image


Also a tiny nit - can you indent the `<li>` elements and have the `</ul>` inline with the opening `<ul>`>

Comment (karlchillmaid) [reply app/templates/views/pricing/index.html:276]: Thanks @samuelhwilliams.

Serves me right for hurriedly copy-pasting the code from last time around.

File: app/templates/views/pricing/index.html [hunk 2]
```diff
@@ -266,4 +266,13 @@ <h2 class="heading-medium" id="letters">Letters</h2>
     {% endcall %}
   </div>
 
+  <div class="panel panel-border-wide">
```
Comment (karlchillmaid) [inline app/templates/views/pricing/index.html:269]:
```suggestion
  <div class="govuk-inset-text"  
```

**Review by samuelhwilliams — APPROVED** [review]
Looks good! 👍

**Pull request merged**